src/helpers/CheckBalanceHelper.ts

```diff
@@ -0,0 +1,15 @@
+import {RunQuery, initConnection, closeConnection} from "../utils/Database.js";
```

Comment:
Remove unnecessary imports (init- and closeConnection are not used in these files)